GenumKit/Parsers/FontsFileParser.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  SwiftGen
+//  Created by Derek Ostrander on 3/7/16.
+//  Copyright (c) 2015 Olivier Halligon
+//  MIT License
+//
+
+import Foundation
+import AppKit.NSFont
+
+// MARK: Font
+
+public struct Font {
+  let familyName: String
+  let style: String
+  let postScriptName: String
+
+  init(familyName: String, style: String, postScriptName: String) {
+    self.familyName = familyName
+    self.style = style
+    self.postScriptName = postScriptName
+  }
+}
```

Comment:
Structure types automatically receive a memberwise initializer like the one you defined) if you don't provide any custom initializer.  You can remove the init.